client/state/README.md

```diff
@@ -49,28 +49,6 @@ All the application information and data in Calypso should go through this data
 
 `state/utils.js` contains a number of helper utilities you may find useful in implementing your state subtree:
 
-### extendAction( action, data )
```

Comment:
Now that we're moving these to packages, should we update the documentation to say that some of the state utilities are in the `state-utils` package, and we're in the process of moving the rest there?